src/mixins/filing-template-mixin.ts

```diff
@@ -42,6 +53,8 @@ export default class FilingTemplateMixin extends Vue {
   @Getter getBusinessContact!: ContactPointIF
   @Getter getAgreementType!: string
   @Getter getOriginalSnapshot: BusinessSnapshotIF[]
+  @Getter getNewResolutionDates!: string[]
+  @Getter getSnapshotShareStructure!: ShareStructureIF
```

Comment:
Object or array?

Reply:
Object

Reply:
That contains some properties and an array.

Reply:
Hmm, looks different than business-snapshot-interface::shareClasses.

Reply:
You betcha, they are different. ShareClassesIF is a sub interface. The ShareStructure is the parent object which the back end would return to us, i'm pretty sure. 

We break down that to share classes as it's a little easier to manage at the component level i think, for iterations and whatnot.

I can have a close look though, maybe something is awry!